backend-api/src/functions/asyncActiveSession/asyncActiveSessionHandler.ts

```diff
@@ -55,20 +56,37 @@ export async function lambdaHandlerConstructor(
     logger.log("INTERNAL_SERVER_ERROR", {
       errorMessage: getSubFromTokenResult.value.errorMessage,
     });
-    return serverError500Response;
+    return serverErrorResponse;
+  }
+
+  const sessionService = dependencies.sessionService(config.SESSION_TABLE_NAME);
+  const getActiveSessionDetailsResult =
```

Comment:
minor (as above) - I'm not keen on 'details' in the name

Reply:
Done